packages/clarity-js/src/data/metadata.ts

```diff
@@ -139,3 +144,47 @@ function cookie(key: string): string {
   }
   return null;
 }
+
+function addPerformanceDimensions(): void {
```

Comment:
We already send this as part of performance module. We should use that data, instead of sending again.  See: https://github.com/microsoft/clarity/blob/master/packages/clarity-js/src/performance/navigation.ts